src/BenchmarkDotNet/Mathematics/OutlierMode.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BenchmarkDotNet.Mathematics
+{
+    public enum OutlierMode
+    {
+        None, OnlyUpper, OnlyLower, All
```

Comment:
I would add some xml docs here (some users might not understand what is Upper and Lower in this context)

Reply:
Done.